crates/biome_js_analyze/src/lint/nursery/use_consistent_member_accessibility.rs

```diff
@@ -0,0 +1,558 @@
+    ///   }
+    ///   // Property is set accessibility modifier
+    ///   private animalName: string; // Property
+    ///   get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   // set accessor is set accessibility modifier
+    ///   set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   // walk() is set accessibility modifier
+    ///   protected walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// ### Valid
+    /// The following patterns are considered correct code with the default options `noPublic`:
+    /// ```jsx,ignore
+    /// class Animal {
+    ///   constructor(
+    ///     public breed,
+    ///     name,
+    ///   ) {
+    ///     // Parameter property and constructor
+    ///     this.animalName = name;
+    ///   }
+    ///   private animalName: string; // Property
+    ///   get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   protected walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// The following patterns are considered correct code with the accessibility set to `explicit`:
+    /// ```jsx,ignore
+    /// class Animal {
+    ///   public constructor(
+    ///     public breed,
+    ///     name,
+    ///   ) {
+    ///     // Parameter property and constructor
+    ///     this.animalName = name;
+    ///   }
+    ///   private animalName: string; // Property
+    ///   public get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   public set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   protected walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// The following patterns are considered correct code with the accessibility set to `none`:
+    /// ```jsx,ignore
+    /// class Animal {
+    ///   constructor(
+    ///     breed,
+    ///     name,
+    ///   ) {
+    ///     // Parameter property and constructor
+    ///     this.name = name;
+    ///   }
+    ///   animalName: string; // Property
+    ///   get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// ## Options
+    ///
+    /// The rule supports the following options:
+    ///
+    /// ```json
+    /// {
+    ///     "//": "...",
+    ///     "options": {
+    ///         "accessibility": "explicit"
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// ### `accessibility`
```

Comment:
Same as above. Let's put blank lines before and after a heading for compatibility